lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -563,24 +565,45 @@ migrateManually tr proxy defaultFieldValues =
     -- which is pretty lame. This was changed later on, but already
     -- serialized data may subsist on for quite a while. Hence this little
     -- pirouette here.
-    renameAccountingStyle :: Sqlite.Connection -> IO ()
-    renameAccountingStyle conn = do
-        renameColumn conn (DBField SeqStateAddressAccountingStyle)
+    renameRoleFields :: Sqlite.Connection -> IO ()
+    renameRoleFields conn = do
+        renameColumnField conn (DBField SeqStateAddressRole)
             "u_tx_o_internal" "utxo_internal"
-        renameColumn conn (DBField SeqStateAddressAccountingStyle)
+        renameColumnField conn (DBField SeqStateAddressRole)
             "u_tx_o_external" "utxo_external"
 
+    -- | Rename column table of SeqStateAddress from 'accounting_style' to `role`
+    -- if needed.
+    renameRoleColumn :: Sqlite.Connection -> IO ()
+    renameRoleColumn conn =
+        isFieldPresent conn roleField >>= \case
+            TableMissing ->
+                traceWith tr $ MsgManualMigrationNotNeeded roleField
+            ColumnMissing -> do
+                traceWith tr $ MsgManualMigrationNotNeeded roleField
```

Comment:
This message is lying.

Reply:
corrected in https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/e3e413b5003318e5827b1d829d0470fd87b54f90

Reply:
That message is still the same though :grimacing: